iati_datastore/iatilib/frontend/api1.py

```diff
@@ -222,7 +222,7 @@ def get(self, format):
 
 
 class DataStoreCSVView(DataStoreView):
-    def get(self, format=".csv"):
+    def get(self, format):
```

Comment:
Note: removal of default due to being superfluous https://github.com/IATI/IATI-Datastore/pull/258#issuecomment-292226970